### PR TITLE
Enable setting to create draft, not live, pages on translation.

### PR DESCRIPTION
**Pull request by mattlinares**
As per #655 

and discussed on Slack: https://wagtailcms.slack.com/archives/CD4S91XF0/p1668602376804759

### CODE REVIEW

File: wagtail_localize/operations.py
Comment (zerolab) [inline wagtail_localize/operations.py:76]: best to keep the setting prefix as `WAGTAILLOCALIZE_` (related: #494)

`WAGTAIL_LOCALIZE_PUBLISH_TRANSLATIONS_OF_LIVE_PAGES` is rather verbose. WAGTAILLOCALIZE_SYNC_LIVE_STATUS_ON_TRANSLATE` may be better and reflect what his happening here.


```python
publish_on_translate_setting = getattr(settings, "WAGTAILLOCALIZE_SYNC_LIVE_STATUS_ON_TRANSLATE", True)

if publish_on_translate_setting:
    publish = getattr(instance, "live", True)
else:
   publish = False
```

Comment (zerolab) [reply wagtail_localize/operations.py:76]: Also the new setting needs documenting in https://github.com/wagtail/wagtail-localize/blob/main/docs/how-to/installation.md

**Review by zerolab — CHANGES_REQUESTED** [review]

File: wagtail_localize/operations.py
Comment (mattlinares) [reply wagtail_localize/operations.py]: @zerolab Done that, thanks!

**Comment by Adrian-Hernando-Noom** [comment]
Any chance this will be merged any time soon? Thank you!

File: wagtail_localize/operations.py
Comment (zerolab) [inline wagtail_localize/operations.py:77]:
```suggestion
            if publish_on_translate_setting:
```

**Comment by mattlinares** [comment]
@zerolab Hopefully this test does the job now. Happy to expand on this if you think it needs more coverage. Thanks.

https://github.com/wagtail/wagtail-localize/pull/656/commits/a606f594261f8bd0980c1f040813397d5dc26c8e

**Comment by zerolab** [comment]
Merged in 7dd46559c2a32324470a166c100fe9fff2909f71. Thanks @mattlinares

**Comment by mattlinares** [comment]
Hi @zerolab this didn't actually get merged in the end because of a Flake8 error I think: https://app.circleci.com/pipelines/github/wagtail/wagtail-localize/2049/workflows/e1a85faf-ba5e-4f4b-80ca-5b513c54aca7/jobs/3607

I have fixed that and PR'd again. Hopefully you can merge this successfully now (though it also seems to have failed checks `This workflow requires approval from a maintainer. 5 failing checks`).

https://github.com/wagtail/wagtail-localize/pull/675

Sorry for the bother!

**Pull request closed without merging**
